### PR TITLE
feat(TMRX-1267): implement slice feedback

### PR DESCRIPTION
**Pull request by douglasmik**
Implementation of slice feedback following design review;
- spacing between grid sections on ContentBucket1
- change from padding to margin for focus boundaries
- add focus to tag
- removal of focus from ArticleList container
- update SliceHeader icon
- updated images with correct ratios
- Focus boundaries on articleList components at larger breakpoints
- theme switch control on ContentBucket slices, to toggle between `TimesWebLightTheme` and `TimesWebLightSportTheme` (can add more/all in the future)

TMRX-1267

### CODE REVIEW

**Review by ibrahim-kurhan — APPROVED** [review]

**Comment by messmike** [comment]
Sorry @douglasmik, additional thing is that the Tag is missing hover and active states (underline styling), as outlined in the [handoff](https://www.figma.com/file/phASjaJaTK2CD5MwZUZlZr/Slices-handoff?type=design&node-id=41%3A95777&t=AvvW8W2szRGw4B0C-1)

**Comment by douglasmik** [comment]
> handoff

@messmike If we set main card link to `expand` we don't get to the different states on the Tag. Please advise

**Comment by messmike** [comment]
Could we have a rule where if a card has a Tag, then the expand prop isn't set, and expand is set to true if no Tag is present?

File: packages/ts-newskit/src/slices/content-bucket-1/content-bucket-1.stories.mdx [hunk 1]
```diff
@@ -25,17 +23,31 @@ This takes in a series of props, as below, to display the part of the slices.
...
+<Story 
+  name="ContentBucket1"
+  args={{
+    section,
+    leadStory,
+    articles,
+    comments
+  }}
+  argTypes={{
+    theme: {
```
Comment (bibicollins1) [inline packages/ts-newskit/src/slices/content-bucket-1/content-bucket-1.stories.mdx:44]: Nice!!

**Review by bibicollins1 — APPROVED** [review]
Looks good!

**Pull request merged**